trace-dispatcher/src/Cardano/Logging/TraceDispatcherMessage.hs

```diff
@@ -147,9 +160,11 @@ instance MetaTrace TraceDispatcherMessage where
     documentFor (Namespace _ ["TracerConsistencyWarnings"]) = Just $ mconcat
       [ "Tracer consistency check found errors."
       ]
+    documentFor (Namespace _ ["InfoConfig"]) = Just $ mconcat
```

Comment:
The name is without `Tracer...` here, yet it provides a `TracerConfig` json. Maybe we could rename it to sth. like `TracerConfigInfo` to keep the pattern? Or is there a reason to set it apart?

Reply:
Changed